frappe/core/doctype/communication/comment.py

```diff
@@ -81,7 +81,9 @@ def notify_mentions(doc):
 			return
 
 		sender_fullname = get_fullname(frappe.session.user)
-		parent_doc_label = "{0} {1}".format(_(doc.reference_doctype), doc.reference_name)
+		title_field = frappe.get_meta(doc.reference_doctype).get_title_field()
+		parent_doc_label = "{0}: {1} (#{2})".format(_(doc.reference_doctype),
```

Comment:
should be an if/else in case there is no title_field

Reply:
@rmehta `get_title_field` will return the `name` if title field is not available